Tests/base/NSFileManager/general.m

```diff
@@ -128,6 +128,11 @@ int main()
     PASS([str1 isEqualToString: str2],"NSFileManager moved file contents match");
   }
 
+  PASS(![mgr copyPath: @"NSFMFile"
+	       toPath: @"NSFMDestination"
+	      handler: nil],
+       "NSFileManager does not copy nonexistent file");
+
```

Comment:
The test isn't checking the error.  It would be good to check that we give the same error (and call the same delegate methods) as Cocoa.